code/src/components/ThoughtInput.css

```diff
@@ -0,0 +1,84 @@
+form {
```

Comment:
Nice, next level, putting the css in a separat part and connecting it to the component! YES!